Spec/Auth.swift

```diff
@@ -1470,6 +1470,7 @@ class Auth : QuickSpec {
                         clientOptions.authMethod = "POST"
                         clientOptions.authHeaders = ["X-Header-1": "foo", "X-Header-2": "bar"]
                         let tokenParams = ARTTokenParams()
+                        tokenParams.timestamp = Date()
```

Comment:
You shouldn't need this. The timestamp should be generated in the auth server, when creating the signed token request

Reply:
@SimonWoolf done e4a4591. Thanks.